calculator.py

```diff
@@ -0,0 +1,21 @@
+class Calculator:
+    def add(self, numbers: str) -> int:
+
+        numbers = numbers.strip()
+        delimiter = ["//", ";", '*', ']', '[', ',', "\n", "%"]
+        for i in delimiter:
+            numbers = numbers.replace(i, " ")
+        numbers = " ".join(numbers.split())
+
+        split_numbers = numbers.split(" ")
+        split_numbers = list(filter(None, split_numbers))
+        split_numbers = list(map(int, split_numbers))
+
+        negative_numbers = [x for x in split_numbers if x < 0]
+
+        if len(negative_numbers) > 0:
```

Comment:
We could simplify this line:
````python
if len(negative_numbers):
````
What do you think?

Reply:
it's wonderful